crates/sui-network/src/randomness/mod.rs

```diff
@@ -343,17 +348,16 @@ impl RandomnessEventLoop {
             debug!("skipping received partial sigs, we already have completed this sig");
             return;
         }
-        let expected_share_count = if let Some(count) = peer_share_counts.get(&peer_id) {
-            count
+        let expected_share_ids = if let Some(expected_share_ids) = peer_share_ids.get(&peer_id) {
+            expected_share_ids
         } else {
-            debug!("received partial sigs from unknown peer");
+            warn!("received partial sigs from unknown peer");
```

Comment:
this would be a local anemo related bug, so if the messages above remain `debug` i think this one should too